capitals.rb

```diff
@@ -5,148 +5,175 @@
+# }, {
+#     name: "Florida",
+#     capital: "Tallahassee"
+# }, {
+#     name: "Georgia",
+#     capital: "Atlanta"
+# }, {
+#     name: "Hawaii",
+#     capital: "Honolulu"
+# }, {
+#     name: "Idaho",
+#     capital: "Boise"
+# }, {
+#     name: "Illinois",
+#     capital: "Springfield"
+# }, {
+#     name: "Indiana",
+#     capital: "Indianapolis"
+# }, {
+#     name: "Iowa",
+#     capital: "Des Moines"
+# }, {
+#     name: "Kansas",
+#     capital: "Topeka"
+# }, {
+#     name: "Kentucky",
+#     capital: "Frankfort"
+# }, {
+#     name: "Louisiana",
+#     capital: "Baton Rouge"
+# }, {
+#     name: "Maine",
+#     capital: "Augusta"
+# }, {
+#     name: "Maryland",
+#     capital: "Annapolis"
+# }, {
+#     name: "Massachusetts",
+#     capital: "Boston"
+# }, {
+#     name: "Michigan",
+#     capital: "Lansing"
+# }, {
+#     name: "Minnesota",
+#     capital: "St. Paul"
+# }, {
+#     name: "Mississippi",
+#     capital: "Jackson"
+# }, {
+#     name: "Missouri",
+#     capital: "Jefferson City"
+# }, {
+#     name: "Montana",
+#     capital: "Helena"
+# }, {
+#     name: "Nebraska",
+#     capital: "Lincoln"
+# }, {
+#     name: "Nevada",
+#     capital: "Carson City"
+# }, {
+#     name: "New Hampshire",
+#     capital: "Concord"
+# }, {
+#     name: "New Jersey",
+#     capital: "Trenton"
+# }, {
+#     name: "New Mexico",
+#     capital: "Santa Fe"
+# }, {
+#     name: "New York",
+#     capital: "Albany"
+# }, {
+#     name: "North Carolina",
+#     capital: "Raleigh"
+# }, {
+#     name: "North Dakota",
+#     capital: "Bismarck"
+# }, {
+#     name: "Ohio",
+#     capital: "Columbus"
+# }, {
+#     name: "Oklahoma",
+#     capital: "Oklahoma City"
+# }, {
+#     name: "Oregon",
+#     capital: "Salem"
+# }, {
+#     name: "Pennsylvania",
+#     capital: "Harrisburg"
+# }, {
+#     name: "Rhode Island",
+#     capital: "Providence"
+# }, {
+#     name: "South Carolina",
+#     capital: "Columbia"
+# }, {
+#     name: "South Dakota",
+#     capital: "Pierre"
+# }, {
+#     name: "Tennessee",
+#     capital: "Nashville"
+# }, {
+#     name: "Texas",
+#     capital: "Austin"
+# }, {
+#     name: "Utah",
+#     capital: "Salt Lake City"
+# }, {
+#     name: "Vermont",
+#     capital: "Montpelier"
+# }, {
+#     name: "Virginia",
+#     capital: "Richmond"
+# }, {
+#     name: "Washington",
+#     capital: "Olympia"
+# }, {
+#     name: "West Virginia",
+#     capital: "Charleston"
+# }, {
+#     name: "Wisconsin",
+#     capital: "Madison"
+# }, {
+#     name: "Wyoming",
+#     capital: "Cheyenne"
+
+states = states.shuffle
+
+states.map do |state|
+  state[:correct] = 0
+  state[:wrong] = 0
+end
+
+while true
+  states.each do |state|
+    puts "what is the capital of: " + state[:name] + "?"
+    guess = gets.chomp
+    if guess == state[:capital]
+      puts "that's right!"
+      state[:correct] +=1
+    else
+      puts "Wrong."
+      state[:wrong] +=1
+    end
+    total = state[:correct] + state[:wrong]
```

Comment:
I would move this out the `.each` block and down to L#178 -- It will display the total for that round